.kokoro/tests/run_tests.sh

```diff
@@ -78,6 +78,8 @@ if [[ "$SCRIPT_DEBUG" != "true" ]]; then
     source "${KOKORO_GFILE_DIR}/firestore_secrets.txt"
     # spellcheck source=src/cts_v4_secrets.txt
     source "${KOKORO_GFILE_DIR}/cts_v4_secrets.txt"
+    # shellcheck source=src/cloud_sql_secrets.txt
+    source "${KOKORO_GFILE_DIR}/cloud_sql_secrets.txt"
```

Comment:
Have you already set this up? If not we can use secret manager instead.

Reply:
I have but it's not working, so let's try using Secret Manager instead

Reply:
sgtm. Dropping this example if you decided to switch https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/.kokoro/build-with-run.sh#L92

Reply:
Got it working using the secrets file in Cloud Devrel Kokoro Resources. I think for now we can keep this setup, but eventually we should start migrating all the secrets to secret manager.